plugins/repository-s3/src/main/java/org/elasticsearch/repositories/s3/S3Repository.java

```diff
@@ -89,7 +89,7 @@
     /**
      * Maximum size of files that can be uploaded using a single upload request.
      */
-    static final ByteSizeValue MAX_FILE_SIZE = new ByteSizeValue(5, ByteSizeUnit.GB);
+    static final ByteSizeValue MAX_FILE_SIZE = new ByteSizeValue(2, ByteSizeUnit.GB);
```

Comment:
This setting is used (though not directly) to define the upper bound of the BUFFER_SIZE_SETTING. Changing it like this might break current installations.

Reply:
@tlrx right, sorry I though I'd put this into team discuss mode this isn't a trivial change. I'll ping you on another channel about this one.